docs/metrics-overview.md

```diff
@@ -29,6 +29,116 @@ The COSI driver exposes default gRPC server metrics to monitor RPC activity.
 grpc_server_started_total{grpc_method="DriverGetInfo",grpc_service="cosi.v1alpha1.Identity",grpc_type="unary"} 2
 ```
 
+---
+
+## IAM Operation Metrics
+
+The COSI driver collects metrics for IAM operations performed via the AWS IAM API. These metrics help track the number and duration of IAM-related operations, enabling better monitoring and observability of IAM activity.
+
+### Key IAM Metrics
+
+| Metric Name                                        | Description                                                | Labels            | Example Values                  |
+|---------------------------------------------------|------------------------------------------------------------|-------------------|----------------------------------|
+| `scality_cosi_driver_iam_request_duration_seconds`| Histogram of IAM request durations in seconds.             | `method`, `status`| `CreateUser`, `success`, `error`|
```

Comment:
You may want to mention only one of `success` or `error` in the example values

Reply:
And in the table below for `IAM Operations` note  that it is the `method` label

And add another table for the `status` label with `success` and `error`

Reply:
changed the doc to change method to action and also add labels table separately.
https://github.com/scality/cosi-driver/pull/85/commits/989ef11ce706ab73b764fb4e338846eec98be67e